content/talks/no-grid-system.md

```diff
@@ -28,11 +28,11 @@ events:
     adr: Düsseldorf, Germany
     date: 2018-05-07
     # slides: https://oddbooksapp.com/release/ce37ef91-8bd8-43e2-932a-66931b4b25ce
-    video: https://vimeo.com/268576559
-    media: &bt
-      iframe: https://player.vimeo.com/video/268576559
-      width: 640
-      height: 360
+    # video: https://vimeo.com/268576559
+    # media: &bt
+    #   iframe: https://player.vimeo.com/video/268576559
+    #   width: 640
+    #   height: 360
```

Comment:
@mirisuzanne It looks like these are no longer available?

Reply:
@jgerigmeyer looks like they moved to youtube

```suggestion
    video: https://youtu.be/iODambgMZhk
    media: &bt
      iframe: https://www.youtube.com/embed/iODambgMZhk
```